src/libraries/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -677,28 +677,17 @@ public static unsafe string Title
 
         public static void Beep()
         {
-            const char BellCharacter = '\u0007'; // Windows doesn't use terminfo, so the codepoint is hardcoded.
-
             if (!Console.IsOutputRedirected)
             {
-                Console.Out.Write(BellCharacter);
```

Comment:
Is there a test we can add that would have failed with the old impl here?

Reply:
I currently don't see a way to properly test it. We just call a sys-call when the output is redirected or perform a direct write when it's not, so I don't see a way to capture this information.

Reply:
You can validate that the beep character is not being written to the TextWriter set with SetOut?